### PR TITLE
Search front end mobile improvements

### LINKED ISSUE #5831: Update explanatory text on search form
Updated text for the search form, to help clarify the coverage of catalogue search and image search.

Catalogue:
- Find thousands of books, manuscripts, visual materials and unpublished archives from our collections, many of them with free online access.

Images:
- Search for free, downloadable images taken from our library and museum collections, including paintings, illustrations, photos and more.

### PR DESCRIPTION
**Pull request by ishouty**
* updating colours "Sort by" to black 
* updating "Sort by" to "Sort by:"
* update text for search collections #5831
* Do not display (paging icons, total records) on the top of search results for mobile devices to reduce clutter on multiple case scenarios e.g 

- multiple paging icons displayed at once 
- total size records display more than two digits 
- sort by - long text selected -> newest to oldest 

closes #5831

### CODE REVIEW

File: common/views/components/Paginator/Paginator.js [hunk 1]
```diff
@@ -97,40 +105,40 @@ const Paginator = ({
...
-              clickHandler={event => {
-                onPageChange(event, next);
-              }}
-              type="light"
-              icon="arrow"
-              text={`Next (page ${next})`}
-            />
-          </Space>
-        )}
+        <PaginatorWrapper hideMobilePaging={hideMobilePaging}>
```
Comment (jamesgorrie) [inline common/views/components/Paginator/Paginator.js:108]: Could we not just use the [`useScreenSize` hook](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/hooks/useWindowSize.js) and remove the DOM completely?

Why would we rather hide it via css?

Comment (ishouty) [reply common/views/components/Paginator/Paginator.js:108]: That is also a solution i could do, I was considering non javascript version as well. Should we leverage on both ?

File: catalogue/webapp/pages/images.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ type ImagesPaginationProps = {
   results: CatalogueResultsList<Image>;
   imagesRouteProps: ImagesRouteProps;
   setSavedSearchState: (state: ImagesRouteProps) => void;
+  hideMobilePaging?: boolean;
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/images.tsx:56]: could we call this `hideMobilePagination` for consistency?

File: common/views/components/Paginator/Paginator.js (hunk 1 above)
Comment (jamesgorrie) [reply common/views/components/Paginator/Paginator.js:108]: Good point, the hook is for JS.

File: catalogue/webapp/pages/images.tsx (hunk 2 above)
Comment (ishouty) [reply catalogue/webapp/pages/images.tsx:56]: yep sounds good

**Review by jamesgorrie — APPROVED** [review]

**Review by gestchild — APPROVED** [review]

**Pull request merged**